### PR TITLE
Update for 0.9-rc.1

### PR DESCRIPTION
**Pull request by cdata**
/cc @morethanreal

### CODE REVIEW

File: iron-resizable-behavior.html [hunk 1]
```diff
@@ -9,24 +9,24 @@
...
+action on their new measurements).
+Elements that perform measurement should add the `IronResizableBehavior` behavior to
+their element definition and listen for the `iron-resize` event on themselves.
+This event will be fired when they become showing after having been hidden,
+when they are resized explicitly by another resizable, or when the window has been
+resized.
+Note, the `iron-resize` event is non-bubbling.
+@group Polymer Behaviors
+@element iron-resizable-behavior
+@homepage github.io
```
Comment (notwaldorf) [inline iron-resizable-behavior.html:26]: i've been removing homepage. should i be adding it back? 😓(this isn't actionable in this PR, just saying)

**Comment by notwaldorf** [comment]
lgtm

File: iron-resizable-behavior.html (hunk 1 above)
Comment (cdata) [inline iron-resizable-behavior.html:26]: I don't know. I don't think we have been very consistent with these annotations.

Comment (morethanreal) [inline iron-resizable-behavior.html:26]: FWIW I've been removing all annotations.

**Pull request merged**